analyzers/its/expected/Ember-MM/EmberAPI-{208AA35E-C6AE-4D2D-A9DD-B6EFD19A4279}-S2259.json

```diff
@@ -2,6 +2,71 @@
 "issues":  [
 {
 "id":  "S2259",
+"message":  "'_ComplexAdvancedSettings.FirstOrDefault(Function(f) f.Name = key AndAlso f.Section = Assembly)' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\Ember-MM\EmberAPI\clsAdvancedSettings.vb",
+"region":  {
+"startLine":  192,
+"startColumn":  17,
+"endLine":  192,
+"endColumn":  111
+}
+}
+},
```

Comment:
Unknown because the type is defined elsewhere, but presumably `TableItem` is a field.
https://github.com/SonarSource/sonar-dotnet/blob/145047ee1ed01316ef730427c7af96f0602ee2d8/analyzers/its/sources\Ember-MM\EmberAPI\clsAdvancedSettings.vb#L192